<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="utf-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        
        
        
        <link rel="shortcut icon" href="../img/favicon.ico">
        <title>   - Break Through Cancer Data Reference</title>
        <link href="../css/bootstrap.min.css" rel="stylesheet">
        <link href="../css/font-awesome.min.css" rel="stylesheet">
        <link href="../css/base.css" rel="stylesheet">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.8.0/styles/github.min.css">
        <link href="../extra.css" rel="stylesheet">
        <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.8.0/highlight.min.js"></script>
        <script>hljs.highlightAll();</script> 
    </head>

    <body>
        <div class="navbar fixed-top navbar-expand-lg navbar-dark bg-primary">
            <div class="container">
                <a class="navbar-brand" href="..">Break Through Cancer Data Reference</a>
                <!-- Expander button -->
                <button type="button" class="navbar-toggler" data-toggle="collapse" data-target="#navbar-collapse">
                    <span class="navbar-toggler-icon"></span>
                </button>

                <!-- Expanded navigation -->
                <div id="navbar-collapse" class="navbar-collapse collapse">
                        <!-- Main navigation -->
                        <ul class="nav navbar-nav">
                            <li class="navitem">
                                <a href=".." class="nav-link">  </a>
                            </li>
                            <li class="navitem">
                                <a href="../faq/" class="nav-link">FAQ</a>
                            </li>
                            <li class="navitem">
                                <a href="../about/" class="nav-link">  </a>
                            </li>
                            <li class="navitem active">
                                <a href="./" class="nav-link">  </a>
                            </li>
                        </ul>

                    <ul class="nav navbar-nav ml-auto">
                        <li class="nav-item">
                            <a href="#" class="nav-link" data-toggle="modal" data-target="#mkdocs_search_modal">
                                <i class="fa fa-search"></i> Search
                            </a>
                        </li>
                            <li class="nav-item">
                                <a rel="prev" href="../about/" class="nav-link">
                                    <i class="fa fa-arrow-left"></i> Previous
                                </a>
                            </li>
                            <li class="nav-item">
                                <a rel="next" class="nav-link disabled">
                                    Next <i class="fa fa-arrow-right"></i>
                                </a>
                            </li>
                    </ul>
                </div>
            </div>
        </div>

        <div class="container">
            <div class="row">
                    <div class="col-md-3"><div class="navbar-light navbar-expand-md bs-sidebar hidden-print affix" role="complementary">
    <div class="navbar-header">
        <button type="button" class="navbar-toggler collapsed" data-toggle="collapse" data-target="#toc-collapse" title="Table of Contents">
            <span class="fa fa-angle-down"></span>
        </button>
    </div>

    
    <div id="toc-collapse" class="navbar-collapse collapse card bg-secondary">
        <ul class="nav flex-column">
            
            <li class="nav-item" data-level="2"><a href="#dash-release-notes" class="nav-link">DASH Release Notes</a>
              <ul class="nav flex-column">
              </ul>
            </li>
            
            <li class="nav-item" data-level="2"><a href="#2025_10_24" class="nav-link">2025_10_24</a>
              <ul class="nav flex-column">
              </ul>
            </li>
            
            <li class="nav-item" data-level="2"><a href="#2025_06_30" class="nav-link">2025_06_30</a>
              <ul class="nav flex-column">
              </ul>
            </li>
            
            <li class="nav-item" data-level="2"><a href="#2025_03_27" class="nav-link">2025_03_27</a>
              <ul class="nav flex-column">
              </ul>
            </li>
            
            <li class="nav-item" data-level="2"><a href="#2024_06_03" class="nav-link">2024_06_03</a>
              <ul class="nav flex-column">
              </ul>
            </li>
            
            <li class="nav-item" data-level="2"><a href="#2024_03_04" class="nav-link">2024_03_04</a>
              <ul class="nav flex-column">
              </ul>
            </li>
        </ul>
    </div>
</div></div>
                    <div class="col-md-9" role="main">

<style>
   .navbar { display: none; }
   .bs-sidebar
</style>

<h2 id="dash-release-notes">DASH Release Notes</h2>
<hr>

<h2 id="2025_10_24"><strong>2025_10_24</strong></h2>
<p>Progress in data science across the BTC portfolio in Q3 has been remarkable.  Notably, <strong>recent publications
highlight BTC investigator-led advances in computational analysis,</strong> including:
the <a href="https://doi.org/10.1038/s41586-025-09580-0">CloneSeq-SV</a> method for tracking clonal evolution using cell-free DNA and single-cell WGS;
the <a href="https://doi.org/10.1038/s41467-025-61142-0">MorphLink</a> framework for co-analyzing cellular morphology and molecular dynamics in spatial omics;
and a preprint describing the <a href="https://doi.org/10.1101/2025.08.23.671943">ProteomeGenerator3</a> tool for integrated analysis of long-read transcriptomics and multi-dimensional proteomics.</p>
<p><strong>Data shared to DASH has swelled to 58 TB, effectively doubling in 9 months</strong> and now spanning more than 1100 subjects and 12K samples.  Reflecting contributions from all active disease TeamLabs and several clinical studies, these data support numerous publications and continue to enhance BTC’s capacity for multi-modal discovery.</p>
<p><strong>Finally, the DASH platform has also advanced significantly,</strong> with a <a href="https://dash.breakthroughcancer.org">faster and more unified portal interface</a>
<img alt="new-DASH-landing-page" src="../img/DASH2.png" />
that showcases major enhancements to existing applications and launches several new ones, including:</p>
<ul>
<li><strong>PipeLand</strong> &nbsp;&nbsp; graphically navigate analysis tools by data modality, with point-click pipeline execution in Cirro</li>
<li><strong>SCRP</strong>  &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; prototype for cross-institutional exploration of single cell RNAseq datasets per TeamLab</li>
<li><strong>WhoDat</strong> &nbsp;&nbsp;&nbsp;&nbsp; easy way to search for BTC collaborators by name, institution, or TeamLab</li>
</ul>
<p>This update to the DASH ecosystem streamlines end-to-end scientific workflows while exposing a broader and richer set of metadata. <strong>For most datasets, key metrics of accrued data are now visible across BTC</strong> and can also be explored on a per-TeamLab basis in both summary form or more granular, study-specific views.</p>
<h2 id="2025_06_30"><strong>2025_06_30</strong></h2>
<p>As Q2 of 2025 concludes, we’re pleased to report accelerating momentum at the interface of data science and disease-focused translational efforts across the BTC network. Since January, data shared to <a href="https://dash.breakthroughcancer.org">DASH</a>
has surged 93% to 54 TB, now representing over 800 subjects and 2,100 biospecimens (including external sources).  This growth reflects increased harmonization across 8 TeamLabs and several clinical trials, supports multiple publications, and expands BTC’s capacity for integrated, multi-modal discovery.</p>
<p>A highlight of the quarter was the 2nd Annual BTC Hackathon, hosted June 9–10 by the Beroukhim Lab at the Broad Institute. With 67 registrants spanning 7 TeamLabs, the event sparked wide-ranging technical creativity and seeded multiple ongoing collaborations. The winning team, Hack2Align, exemplified the collaborative BTC 
<img alt="hackathon-2-winners" src="../img/hackathon-2-winners.png" />
spirit: drawing on
on the CODA/PIVOT tools originally developed within the DPT TeamLab, four imaging modalities from GBM TeamLab (H&amp;E, Xenium, MALDI, CyCIF/ImF), and analytical expertise from Data Science TeamLab, they developed a prototype workflow for spatial co-registration across imaging modalities. Despite its early-stage nature, the pipeline achieved a surprisingly robust ~40-pixel alignment RMSE and shows potential for enabling more integrated, multimodal analyses across BTC’s spatially resolved datasets.</p>
<h2 id="2025_03_27"><strong>2025_03_27</strong></h2>
<p>Welcome to the first installment of DASH release notes in 2025! The Data Science
Team (DST) closed 2024 with enormous progress and a strong sense of momentum across
BTC, with DASH seeing a
<a href="https://board.breakthroughcancer.org" rel="noopener noreferrer" target="_blank">28X increase in volume of data shared</a>
by TeamLabs, nearly
<a href="https://breakthroughcancer.cirro.bio" rel="noopener noreferrer" target="_blank">200 computational pipelines available in Cirro</a>,
and deeply resourced multimodal analyses underway in numerous TeamLabs&ndash;fueled by novel BTC methods.  In just the first few months
of 2025 the volume of data shared to DASH has grown 42% to over 40TB, reflecting contributions from 8 TeamLabs and notably 
including hundreds of thousands of images from our newest TeamLab, LungRD.
<img alt="DASH-counts-2025-03" src="../img/dash-counts-2025-03-26.png" /> This is <strong>catalyzing innovative scientific engagement</strong> with disease TeamLabs, including collaborations with:</p>
<ul>
<li><strong>The GBM team</strong>, helping advance interpretation of single cell RNA sequencing (scRNA-seq), DNA sequencing, and both T-cell and malignant cell clonal tracking data from the needle biopsy trial, to gain deeper insights into cancer evolution; and spearheading data deposition for the trial pilot publication</li>
<li><strong>The Ovarian MRD team</strong>, to adapt and optimize an approach for longitudinal tracking of drug resistance and co-leading spatial tumor microenvironment profiling to advance our understanding of MRD biology</li>
<li><strong>The KRAS team</strong>, to advance understanding of cellular and molecular mechanisms of response and resistance to KRAS inhibitors by analyzing scRNA-seq data from clinical specimens and mouse models</li>
<li>And contributing to the <strong>evaluation and use of spatial omics platforms</strong> by analyzing and integrating spatial pilot data through the development and application of novel spatial methods.</li>
</ul>
<p><strong>Another important milestone in multimodal analytics is the GBM Explorer</strong>. Developed by Alex Ling (Chiocca Lab, BWH), this APP brings together <strong>5 data modalities</strong> (scRNA, TCR, bulkRNA, images, and clinical annotations), enabling clinicians, computational biologists, and engineers to: overlay experimentally measured values on brain 
<img alt="GBM-EXPLORER" src="../img/gbm-explorer.png" />
biopsy locations in 3D; for rapid co-visualization of measured values from different assays; and easy visualization of patient treatment histories.  Thanks to innovative security infrastructure work by Henry Dewhurst at BTC Central, GBM Explorer is the <strong>first multimodal analysis APP developed completely within BTC</strong> for use across all institutions by members of a single TeamLab, and we look forward to re-using this model to deploy BTC-wide portals for scRNA-seq, scTCR-seq, scBCR-seq and spatial 'omics as well as extend the use of <a href="https://www.cbioportal.org">cBioPortal</a> in BTC.</p>
<h2 id="2024_06_03"><strong>2024_06_03</strong></h2>
<p>In this quarter's installment of the DASH release notes we are happy to again report tremendous progress.</p>
<ul>
<li>An <strong>incredibly successful Hackathon event,</strong> with some 50 registrants across 5 TeamLabs coming together
May 20-21 at MSKCC for: 6 training workshops on emerging scRNA, spatial, TCR, and bulkDNA+RNA pipelines
in Cirro, coupled with hands-on data analysis and real-time tool construction to explore scientific hypotheses.
Each of the working groups performed valuable pipeline, analysis, and exploratory data work; with the winning team
taking home a $150 prize for developing a novel approach to using the pool of doublet cells in GBM single-cell
RNA data (about 10% of cell population) as markers for strong cell-cell interaction, confirming one such 
interaction in matched GBM spatial data, then using L-R gene vector analysis to identify a marked 
gradient effect in a separate, public GBM dataset.  Finally, we'd like to thank the MSKCC and BTC organizing,
events, and Program Management teams for tremendous catalyzing effort and helping create the sense that this
inaugural hackathon is but the first of many to come.</li>
<li>The <strong>scRNA, spatial, and TCR development teams have released multiple versions of their pipelines,</strong> applied 
in conjunction with disease TeamLabs to emerging data.</li>
<li>By way of 6 internal data releases, the <strong>total quantity of data shared to DASH has more than
doubled since March</strong> and number of files nearly tripled; the current <strong>2024-05-21</strong> data version
includes over 20TB across some 1200 samples:
<img alt="DASH-counts-2024-06" src="../img/dash-counts-2024-06-03.png" /></li>
<li>Thanks to Henry Dewhurst and the Cirro team (especially Nathan Thorpe), <strong>we now have an operational, BTC-specific
Cirro tenant.</strong> This provides additional security sandboxing and configuration flexibility, was used in the 
aforementioned hackathon, and is the <strong>final major step towards production-grade, multi-modal analyses workspaces
for each disease TeamLab</strong> to be rolled out in Q2 through Q4 of 2024.</li>
<li><strong>Multiple software releases of the DASH Board, Browse and Curate APPs</strong><ul>
<li>Reflecting new metadata export, annotations, and filtering capabilities</li>
<li>Improved UI, phone home, and metadata collection, towards self-service by data coordinators</li>
</ul>
</li>
</ul>
<h2 id="2024_03_04"><strong>2024_03_04</strong></h2>
<p>Welcome to the inaugural edition of the DASH Release Notes, through which we'll provide regular, BTC-wide updates as new datasets, analysis methods &amp; pipelines, infrastructure, and more emerge.  With this first installment we're happy to relay:</p>
<ul>
<li>A meeting of the Data Science TeamLab (DST) last week at MD Anderson, with thanks to Kadir Akdemir &amp; MDA colleagues for hosting.  This 2-day event showcased considerable progress in the development of analysis methods &amp; pipelines (notably single-cell and bulkRNA, TCR, spatial, and classical bulkDNA approaches); and how they're being woven into an increasingly capable data analysis system for the entire BTC portfolio.  Leading to energetic discussion of future science, cross-team interaction, and trainee development.</li>
<li>
<p>Data snapshot version <strong>2024-02-20</strong>, with over 1K samples and 9TB of data across several TeamLabs:</p>
<ul>
<li>OV-IOC:<ul>
<li>335 STIC histoPath images from the Shih Lab @ JHU (courtesy of Yen-Wei Chien)</li>
<li>Over 200 files of methylation data from the Cope Lab at JHU</li>
</ul>
</li>
<li>Pancreatic: over 450 files including<ul>
<li>Flow, RNASeq, and path images of MRTX1133-treated mice &amp; cell lines from Dougan Lab @ DFCI (courtesy of Li Qiang)</li>
<li>Visium spatialTranscriptomics mouse data from Kalluri Lab @ MDA (courtesy of Kate McAndrews)</li>
<li>Cell surface proteomics from Oni Lab @ MIT (courtesy of Salome Shubitidze)</li>
</ul>
</li>
<li>
<p>More than 4,400 files of GBM data comprising 101 samples and 8 assay data types</p>
<p><em>Contact the <a href="mailto:dash@breakthroughcancer.org">DASH team</a> if you still do not have an account for accessing or analyzing data, but please keep in mind that until embargo periods have passed the data created by a TeamLab will generally be accessible only to members of that TeamLab.</em></p>
</li>
<li>
<p>Data shared to DASH is summarized for easy perusal in the
<a href="https://board.breakthroughcancer.org">Board app</a> ...
<img alt="DASH-Board" src="../img/dash-board.png" />
... may be browsed/downloaded from the <a href="https://data.breakthroughcancer.org">Browse app</a> or
programmatically (e.g. see
<a href="https://cloud.google.com/sdk/gcloud/reference/storage">here</a> or
<a href="https://cloud.google.com/storage/docs/gsutil">here</a>)
<img alt="DASH-Board" src="../img/dash-browse.png" />
... analyzed with any of the 20+ pipeline configurations available in <a href="https://cirro.bio">Cirro</a>
<img alt="Cirro" src="../img/cirro-aims.png" />
... with all easily accessed from the DASH home page at
<a href="https://dash.breakthroughcancer.org">dash.breakthroughcancer.org</a>
<img alt="DASH-HomePage" src="../img/dash-home.png" /></p>
</li>
</ul>
</li>
</ul></div>
            </div>
        </div>

        <footer class="col-md-12">
            <hr>
            <p>Documentation built with <a href="https://www.mkdocs.org/">MkDocs</a>.</p>
        </footer>
        <script src="../js/jquery-3.6.0.min.js"></script>
        <script src="../js/bootstrap.min.js"></script>
        <script>
            var base_url = "..",
                shortcuts = {"help": 191, "next": 78, "previous": 80, "search": 83};
        </script>
        <script src="../js/base.js"></script>
        <script src="../search/main.js"></script>

        <div class="modal" id="mkdocs_search_modal" tabindex="-1" role="dialog" aria-labelledby="searchModalLabel" aria-hidden="true">
    <div class="modal-dialog modal-lg">
        <div class="modal-content">
            <div class="modal-header">
                <h4 class="modal-title" id="searchModalLabel">Search</h4>
                <button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
            </div>
            <div class="modal-body">
                <p>From here you can search these documents. Enter your search terms below.</p>
                <form>
                    <div class="form-group">
                        <input type="search" class="form-control" placeholder="Search..." id="mkdocs-search-query" title="Type search term here">
                    </div>
                </form>
                <div id="mkdocs-search-results" data-no-results-text="No results found"></div>
            </div>
            <div class="modal-footer">
            </div>
        </div>
    </div>
</div><div class="modal" id="mkdocs_keyboard_modal" tabindex="-1" role="dialog" aria-labelledby="keyboardModalLabel" aria-hidden="true">
    <div class="modal-dialog">
        <div class="modal-content">
            <div class="modal-header">
                <h4 class="modal-title" id="keyboardModalLabel">Keyboard Shortcuts</h4>
                <button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
            </div>
            <div class="modal-body">
              <table class="table">
                <thead>
                  <tr>
                    <th style="width: 20%;">Keys</th>
                    <th>Action</th>
                  </tr>
                </thead>
                <tbody>
                  <tr>
                    <td class="help shortcut"><kbd>?</kbd></td>
                    <td>Open this help</td>
                  </tr>
                  <tr>
                    <td class="next shortcut"><kbd>n</kbd></td>
                    <td>Next page</td>
                  </tr>
                  <tr>
                    <td class="prev shortcut"><kbd>p</kbd></td>
                    <td>Previous page</td>
                  </tr>
                  <tr>
                    <td class="search shortcut"><kbd>s</kbd></td>
                    <td>Search</td>
                  </tr>
                </tbody>
              </table>
            </div>
            <div class="modal-footer">
            </div>
        </div>
    </div>
</div>

    </body>
</html>
